content/enterprise/requirements/data-storage/vault.mdx

```diff
@@ -39,45 +39,71 @@ Terraform Enterprise.
 1. Create the `tfe-policy.hcl` file with the following content:
 
   ```hcl
-  path "auth/approle/login" {
-    capabilities = ["create", "read"]
+  # To renew leases.
+  path "sys/leases/renew/*" {
+    policy = "write"
   }
-
   path "sys/renew/*" {
     policy = "write"
   }
 
+  # To renew tokens.
   path "auth/token/renew/*" {
     policy = "write"
   }
 
-  path "transit/encrypt/atlas_*" {
+  # To perform a login.
+  path "auth/approle/login" {
+    capabilities = ["create", "read"]
+  }
+
+  # To upsert transit keys used for key generation.
+  path "transit/keys/atlas_*" {
+   capabilities = ["create", "update"]
+  }
```

Comment:
This is new to support Atlas keys. It matches the policy from our internally-managed Vault.